### PR TITLE
add some missing offsets (more to follow)

### PR DESCRIPTION
**Pull request by fre4kyC0de**
Add support for these builds
- 6.1.7600.20890 (Windows 7 with KB2479710)
- 6.1.7600.21316 (Windows 7 with KB2750090)
- 6.1.7601.21650 (Windows 7 SP1 with KB2479710)
- 6.1.7601.21866 (Windows 7 SP1 with KB2647409)
- 6.1.7601.22104 (Windows 7 SP1 with KB2750090)
- 6.3.9600.19093 (Windows 8.1 with KB4343891)
- 10.0.14393.2457 (Windows 10 RS1)
- 10.0.15063.1155 (Windows 10 RS2)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -331,6 +337,7 @@ Visit [issues](https://github.com/stascorp/rdpwrap/issues) section, and check wh
 - 10.0.14393.0 (Windows 10 RS1 Release 160715-1616)
 - 10.0.14393.1198 (Windows 10 RS1 Release Sec 170427-1353 with KB4019472)
 - 10.0.14393.1737 (Windows 10 RS1 Release Inmarket 170914-1249 with KB4041691)
+- 10.0.14393.2457 (Windows 10 RS1 with 2018-09 rollup; just x64 yet)
```
Comment (binarymaster) [inline README.md]: > `just x64 yet`

Please don't add technical comments to readme.

And add exact build name.

`KB4346783` (from #542)

File: README.md [hunk 2]
```diff
@@ -301,6 +306,7 @@ Visit [issues](https://github.com/stascorp/rdpwrap/issues) section, and check wh
 - 6.3.9600.18692 (Windows 8.1 with KB4022720)
 - 6.3.9600.18708 (Windows 8.1 with KB4025335)
 - 6.3.9600.18928 (Windows 8.1 with KB4088876)
+- 6.3.9600.19093 (Windows 8.1 with 2018-09 rollup)
```
Comment (binarymaster) [inline README.md]: `(Windows 8.1 with KB4343891)`

File: README.md [hunk 3]
```diff
@@ -282,7 +282,12 @@ Visit [issues](https://github.com/stascorp/rdpwrap/issues) section, and check wh
 - 6.0.6002.23521 (Windows Vista SP2 with KB3003743 LDR)
 - 6.1.X.X (Windows 7 / Server 2008 R2)
 - 6.1.7600.16385 (Windows 7)
+- 6.1.7600.20890
+- 6.1.7600.21316
 - 6.1.7601.17514 (Windows 7 SP1)
+- 6.1.7601.21650
+- 6.1.7601.21866
+- 6.1.7601.22104
```
Comment (binarymaster) [inline README.md]: 6.1.7600.20890 (Windows 7 with KB2479710)
6.1.7600.21316 (Windows 7 with KB2750090)
6.1.7601.21650 (Windows 7 SP1 with KB2479710)
6.1.7601.21866 (Windows 7 SP1 with KB2647409)
6.1.7601.22104 (Windows 7 SP1 with KB2750090)

File: res/rdpwrap-ini-kb.txt [hunk 4]
```diff
@@ -343,6 +343,34 @@ DefPolicyPatch.x64=1
 DefPolicyOffset.x64=17AD2
 DefPolicyCode.x64=CDefPolicy_Query_eax_rdi
 
+[6.1.7600.20890]
```
Comment (binarymaster) [inline res/rdpwrap-ini-kb.txt]: KB ini file should include annotations, see nearby sections for the reference.

File: technical.txt [hunk 5]
```diff
@@ -56,6 +56,7 @@ Terminal Services supported versions
 6.3.9600.18692   (Windows 8.1 with KB4022720)                   [init hook + extended patch]
 6.3.9600.18708   (Windows 8.1 with KB4025335)                   [init hook + extended patch]
 6.3.9600.18928   (Windows 8.1 with KB4088876)                   [init hook + extended patch]
+6.3.9600.19093   (Windows 8.1 with 2018-09 rollup)              [init hook + extended patch]
```
[the remark this answers is not in the document]
Comment (fre4kyC0de) [reply technical.txt]: As the DLLs are shipped with both monthly updates (the full rollup and the security-only one) I think, it's better to use the naming scheme I used above [else I would have to list two KB-numbers].

**Comment by fre4kyC0de** [comment]
Requested changes applied.
All data is generated automatically on an Raspberry Pi (using the Windows 10 SDK) and automatically tested using VirtualBox. The output of the scripts and tools are different from the ones tools such as IDA. If I have to do the format stuff (such as rdpwrap-ini-kb.txt) I'd have to do this between 11:00 PM and 1:00 AM (such as now). I'm not able to do this for all builds (not enough free time). I'll provide new offsets at https://github.com/fre4kyC0de/rdpwrap, but just the offsets.

File: technical.txt [hunk 6]
```diff
@@ -16,7 +16,7 @@ Terminal Services supported versions
 6.0.6002.22269   (Windows Vista SP2 with KB977541)              [todo]
 6.0.6002.22340   (Windows Vista SP2 with KB970911)              [todo]
 6.0.6002.22515   (Windows Vista SP2 with KB2381675)             [todo]
-6.0.6002.22641   (Windows Vista SP2 with KB2523307)             [todo]
+6.0.6002.22641   (Windows Vista SP2 with KB2523307)             [unsupported]
```
Comment (binarymaster) [inline technical.txt]: Why unsupported?

Comment (fre4kyC0de) [reply technical.txt]: Cause my script had a bug making it fail on 6.0.6002.22641...
Fixed now

**Pull request merged**